packages/cmf-cqrs/src/middleware/smartWebsocket.js

```diff
@@ -79,6 +80,18 @@ export function startWebsocket(url, offlinebuffer, options) {
 		}
 		ws.close();
 	};
+
+	function wsStillNotYetOpened() {
+		if (ws.readyState === WebSocket.CONNECTING) {
+			ws.onpingtimeout();
+		}
+		clearInterval(wsCreationIntervalId);
```

Comment:
1. you init an interval to notify if the ws is still in connecting state
2. **on the first interval call**, you notify if needed, but in all cases, you clear the interval.
I think you forgot an `else`

Reply:
If the ws is not in connecting state, the interval becomes useless.
If the ws is in connecting mode, the interval is useless as as ws.onpingtimeout will close teh ws and that another is reopen by design of smartwebsocket.

I will indeed change setInterval to setTimeout